homeassistant/components/auth/__init__.py

```diff
@@ -412,4 +412,6 @@ def websocket_current_user(hass, connection, msg):
         'id': user.id,
         'name': user.name,
         'is_owner': user.is_owner,
+        'credentials': [{'auth_provider_type': c.auth_provider_type,
+                         'id': c.id} for c in user.credentials]
```

Comment:
Any usage for c.id?

Reply:
No, just from balloobs comment in chat. I'll remove it and it can be added later if needed

Reply:
I think he meant to `auth_provider_id`, not `c.id`